### PR TITLE
V1.12.15 Updates

### PR DESCRIPTION
**Pull request by ClutchplateDude**
- Corrected ALT calculation for OAM
- Allowed overriding AZ_CIRCUMFERENCE in local config
- Allowed overriding the maximum search distance for autohoming in local config, default remains 30degs

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+**V1.12.15 - Updates**
+- Corrected ALT calculation for OAM
+- Allowed overriding the maximum search distance for autohoming in local config, default remains 30degs
```
Comment (julianneswinoga) [inline Changelog.md:4]: Add something about `AZ_CIRCUMFERENCE`

File: Configuration_adv.hpp [hunk 2]
```diff
@@ -366,24 +369,34 @@
...
 
-    #if AUTOPA_VERSION == 1
-        // the ratio of the ALT gearbox for AutoPA V1 (40:3)
-        #define ALT_WORMGEAR_RATIO (40.0f / 3.0f)
+    #ifdef OAM
+        #ifndef ALT_ROD_PITCH
+            #define ALT_ROD_PITCH 1.0  // mm/rev
+        #endif
+        // the Circumference of the AZ rotation. 209.1mm radius.
+        #define ALT_CIRCUMFERENCE 209.1 * 2 * PI
```
Comment (julianneswinoga) [inline Configuration_adv.hpp:381]: Should `ALT_CIRCUMFERENCE` allowed to be overridden as well?

**Review by julianneswinoga — CHANGES_REQUESTED** [review]

File: Configuration_adv.hpp (hunk 2 above)
Comment (ClutchplateDude) [reply Configuration_adv.hpp:381]: No, not really, since it is fixed by the OAM design whereas the AZ design varies by tripod

Comment (julianneswinoga) [reply Configuration_adv.hpp:381]: Alright I see

**Review by julianneswinoga — APPROVED** [review]

**Pull request merged**